lib/api_browser/app/views/types/home/hash.html

```diff
@@ -0,0 +1,23 @@
+<div ng-if="error" class="alert alert-danger">
+  <p>The requested type could not be found.</p>
+</div>
+<div ng-if="type">
+  <div class="row">
+    <div class="col-lg-12">
+      <h1>
+        {{ type.name | resourceName }}
+      </h1>
+    </div>
+  </div>
+  <div class="row" ng-if="controllers.length">
+    <div class="col-lg-12">
+      <h3>Served by</h3>
+      <ul>
+        <li ng-repeat="controller in controllers">
+          <a ui-sref="root.controller({version: apiVersion, controller: controller.id})">{{ controller.name | resourceName }}</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <ng-include src="'views/type/details.html'" />
```

Comment:
I think the details view could be merged in to the default and given that this is identical to the default, we could get rid of this as well.